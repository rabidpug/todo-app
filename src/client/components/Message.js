// @flow

import React from 'react';

type Props = {
  message: string,
};
const Message: Function = ( { message, }: Props ) => <p>{message}</p>;

export default Message;
