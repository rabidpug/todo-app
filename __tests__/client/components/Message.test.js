import Message from 'Components/Message';
import React from 'react';
import { shallow, } from 'enzyme';
import { shallowToJson, } from 'enzyme-to-json';

describe(
  'Message', () => {
    it(
      'Should render the Message', () => {
        const wrapper = shallow( <Message /> );

        expect( shallowToJson( wrapper ) ).toMatchSnapshot();
      }
    );
  }
);
