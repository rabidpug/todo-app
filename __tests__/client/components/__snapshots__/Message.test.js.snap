// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Should render the Message 1`] = `<p />`;
